packages/connect/src/base/entry.ts

```diff
@@ -121,10 +127,10 @@ export class EntryNodes extends EventEmitter {
   }
 
   /**
-   * @returns a list of entry nodes that are currently used
+   * @returns a list of entry nodes that are currently used (as circuit addresses with us)
    */
   public getUsedRelays() {
```

Comment:
The naming of the function is misleading, since it doesn't return used relay objects but only the circuit addresses.